UnityProject/Assets/Scripts/Systems/InGameEvents/InGameEventScripts/EventSickness.cs

```diff
@@ -31,14 +32,32 @@ public override void OnEventEndTimed()
 
 		private static void SpawnSickness(string serializedEventParameters)
 		{
-			SicknessEventParameters sicknessEventParameters = JsonConvert.DeserializeObject<SicknessEventParameters>(serializedEventParameters);
+			SicknessEventParameters sicknessEventParameters;
+
+			//If null means its a server random event so has no default parameters
+			//Todo: when infection system is more deadly change random effects based on player numbers
```

Comment:
very stupid but can you changet these to ``TODO:`` in all caps. Some IDE's detect them only when they are formatted like that.